pkg/transformers/message.go

```diff
@@ -1,4 +1,4 @@
-package senml
+package transformers
```

Comment:
I am not sure that `Message{}` should be in `transformers` package.

Maybe it should not be in `senml` neither, but I am really skeptical about this one.

Reply:
Let's think about. This is more a DB message format.